lib/galaxy/webapps/galaxy/api/history_contents.py

```diff
@@ -587,7 +587,7 @@ def create(
         ),
         serialization_params: SerializationParams = Depends(query_serialization_params),
         payload: CreateHistoryContentPayload = Body(...),
-    ) -> AnyHistoryContentItem:
+    ) -> Union[AnyHistoryContentItem, List[AnyHistoryContentItem]]:
```

Comment:
Good catch! I think this should be also propagated to the service layer.

Reply:
thanks, i've rebased that into the first commit